server/repositories/RawCaptureRepository.js

```diff
@@ -109,6 +219,17 @@ class RawCaptureRepository extends BaseRepository {
 
     return result[0];
   }
+
+  async update(rawCapture) {
+    const result = await this._session
+      .getDB()
+      .update('status', rawCapture.status)
```

Comment:
Seems like you are updating only the status, I think we should make that filter through the joi schema and this update function should remain 'generic'